### PR TITLE
update rxjs

### PR DESCRIPTION
**Pull request by NemTechCN**
1.update promise to rxjs;
2. update stylesheet.

### CODE REVIEW

File: src/common/vue/collection-record/CollectionRecordTs.ts [hunk 1]
```diff
@@ -135,40 +139,38 @@ export class CollectionRecordTs extends Vue {
...
+            // get transaction by choose recript tx or send
+            if (transactionType == 1) {
                 transferTransaction.forEach((item) => {
-                    if (!item.isReceipt) {
+                    if (item.isReceipt) {
                         list.push(item)
                     }
                 })
                 that.localConfirmedTransactions = list
                 await that.getBlockInfoByTransactionList(that.localConfirmedTransactions, node)
```
Comment (decentraliser) [inline src/common/vue/collection-record/CollectionRecordTs.ts]: Try catch in this block

File: src/common/vue/collection-record/CollectionRecordTs.ts [hunk 2]
```diff
@@ -135,40 +139,38 @@ export class CollectionRecordTs extends Vue {
...
-                    return
-                }
-
+            },
+            node,
+        ).subscribe(async (transactionsInfo) => {
+            let transferTransaction = formatTransactions(transactionsInfo, accountAddress, currentXEM1)
+            let list = []
+            // get transaction by choose recript tx or send
+            if (transactionType == 1) {
```
Comment (decentraliser) [inline src/common/vue/collection-record/CollectionRecordTs.ts]: transactionType == 1 is not very clear, use a constant like transactionType.RECEIVED

File: src/common/vue/collection-record/CollectionRecordTs.ts [hunk 3]
```diff
@@ -135,40 +139,38 @@ export class CollectionRecordTs extends Vue {
...
-                    that.onCurrentMonthChange()
-                    that.isLoadingTransactionRecord = false
-                    return
-                }
-
+            },
+            node,
+        ).subscribe(async (transactionsInfo) => {
+            let transferTransaction = formatTransactions(transactionsInfo, accountAddress, currentXEM1)
+            let list = []
```
Comment (decentraliser) [inline src/common/vue/collection-record/CollectionRecordTs.ts]: Try to never declare variables with `let` and modify them afterwards.

here you can remove `list` and do something like:
 `that.localConfirmedTransactions = transferTransaction.filter(item=>item.isReceipt)`

File: src/common/vue/collection-record/CollectionRecordTs.ts [hunk 4]
```diff
@@ -135,40 +139,38 @@ export class CollectionRecordTs extends Vue {
...
                 that.isLoadingTransactionRecord = false
+                return
+            }
+
+            transferTransaction.forEach((item) => {
+                if (!item.isReceipt) {
+                    list.push(item)
+                }
             })
+            that.localConfirmedTransactions = list
```
Comment (decentraliser) [inline src/common/vue/collection-record/CollectionRecordTs.ts]: same comment about `list`

File: src/common/vue/collection-record/CollectionRecordTs.ts (hunk 1 above)
Comment (NemTechCN) [reply src/common/vue/collection-record/CollectionRecordTs.ts]: fix

File: src/common/vue/collection-record/CollectionRecordTs.ts (hunk 2 above)
Comment (NemTechCN) [reply src/common/vue/collection-record/CollectionRecordTs.ts]: fix

File: src/common/vue/collection-record/CollectionRecordTs.ts (hunk 3 above)
Comment (NemTechCN) [reply src/common/vue/collection-record/CollectionRecordTs.ts]: fix

**Pull request merged**